Assets/Scripts/NPC/NpcOrdering.cs

```diff
@@ -29,17 +33,19 @@ public void DeliverOrder(Dish target)
             if(_hasOrder)
                 return;
 
-            if (target == dishToOrder.Orders[_currentOrder])
+            if (target == dishToOrder.Orders[_currentOrder]){}
```

Comment:
```suggestion
            if (target == dishToOrder.Orders[_currentOrder])
```
Waarom maak je die if leeg?